samples/cpp/continuous_batching_accuracy/continuous_batching_accuracy.cpp

```diff
@@ -61,26 +70,44 @@ int main(int argc, char* argv[]) try {
     std::vector<ov::genai::GenerationConfig> sampling_params(num_prompts);
 
     for (size_t request_id = 0; request_id < num_prompts; ++request_id) {
-        prompts[request_id] = prompt_examples[request_id % prompt_examples.size()];
+        prompts[request_id] = use_prefix ? prefix_str + prompt_examples[request_id % prompt_examples.size()]
+                                         : prompt_examples[request_id % prompt_examples.size()];
         sampling_params[request_id] = sampling_params_examples[request_id % sampling_params_examples.size()];
     }
 
     // Perform the inference
-    
+    auto get_default_block_size = [](const std::string& device) {
+        const size_t cpu_block_size = 32;
+        const size_t gpu_block_size = 16;
```

Comment:
@sshlyapn, where is this parameter set in vanilla Genai API if the benchmark is not used? It should be used by default in GenAI CB pipeline; otherwise, it should always be set by the user, which is not convenient.

Reply:
in future PRs we will hide most of SchedulerConfig from the users and keep only cache eviction config and KV cache size (optionally).